tests/tools/test_create_type.py

```diff
@@ -28,7 +105,7 @@ def get_name() -> str:
     name = strawberry.field(name="name", resolver=get_name)
 
     MyType = create_type("MyType", [name])
-    definition = MyType.__strawberry_definition__
+    definition = get_object_definition(MyType, strict=True)
```

Comment:
With the suggestion this could be done directly without requiring the `get_object_definition` and type-safe

Reply:
I think I prefer using get_object_definition 🍡